### PR TITLE
Fixed video charts to use average incomplete views.

### PR DESCRIPTION
**Pull request by dsjen**
@dan-f

### CODE REVIEW

File: analytics_dashboard/courses/tests/test_presenters.py [hunk 1]
```diff
@@ -233,7 +233,7 @@ def test_attach_aggregated_data_to_parent(self):
             'users_at_start': 60,
             'users_at_end': 40,
             'index': 1,
-            'average_users_at_start': 30,
+            'average_start_only_users': 10,
```
Comment (dan-f) [inline analytics_dashboard/courses/tests/test_presenters.py:236]: Why did you need to update the number here?

**Comment by dan-f** [comment]
:+1:

File: analytics_dashboard/courses/tests/test_presenters.py (hunk 1 above)
Comment (dsjen) [inline analytics_dashboard/courses/tests/test_presenters.py:236]: The test was catching the average number of users at the start of the video, rather than the number of users that _only_ started the video.  We calculate that by the difference of users that ended and started.  It's not exactly accurate, but gives a sense at the least.
